src/helics/apps/helicsWebServer.cpp

```diff
@@ -870,6 +881,33 @@ static const Json::Value null;
 
 namespace helics {
 namespace apps {
+
+    void WebServer::processArgs(const std::string& args)
+
+    {
+        CLI::App parser("http web server parser");
+        parser.allow_extras();
+        parser.add_option("--http_port", httpPort_, "specify the http port to use")
+            ->envname("HELICS_HTTP_PORT");
+        parser.add_option("--http_interface",
+                          httpAddress_,
+                          "specify the interface to use for connecting an http server");
```

Comment:
```suggestion
                          "specify the interface to use for connecting a http server");
```

Reply:
I think this is supposed to be `an` since you are saying the letter which starts with a vowel sound

Reply:
Yea, I think that's right for the most common pronunciation.